tests/test_schema_errors/test_arrays.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+
+from openapi_tester import DocumentationError, SchemaTester
+
+example_schema_array = {"type": "array", "items": {"type": "string"}}
+example_array = ["string"]
+
+tester = SchemaTester()
+
+
+def test_nothing_wrong():
+    """ This should always pass """
+    tester.test_schema_section(example_schema_array, example_array, "")
+
+
+def test_empty_list():
+    """ An empty array should always pass """
+    tester.test_schema_section(example_schema_array, [], "")
+
+
+def test_null():
+    """ A null value should always raise an error """
+    with pytest.raises(DocumentationError, match="Mismatched types, expected list but received NoneType"):
+        tester.test_schema_section(example_schema_array, None, reference="")
+
+
+def test_nullable():
+    """ A null value is allowed when the array is nullable """
+    schema = {"type": "array", "items": {"type": "string"}, "nullable": True}
+    tester.test_schema_section(schema, None, reference="")
```

Comment:
reference should be made an optional param instead of being passed for every test. Just give it a default of "" in the method.

Reply:
Done